### PR TITLE
Add client memory limits docs.

### PR DESCRIPTION
**Pull request by shibd**
### Preview

image

image

### Relation PR
This PR adds doc for https://github.com/apache/pulsar-client-go/pull/991#issuecomment-1512391159

### Documentation

- [x] `doc` 
- [ ] `doc-required` 
- [ ] `doc-not-needed` 
- [ ] `doc-complete`

### CODE REVIEW

File: docs/client-libraries-clients.md [hunk 1]
```diff
@@ -51,4 +51,36 @@ The following example creates a Python client using multiple advertised listener
 
   </TabItem>
 </Tabs>
+````
+
+## Set memory limits
+
+You can uses memory limits param to control client overall memory usage,
```
Comment (Anonymitaet) [inline docs/client-libraries-clients.md:58]:
```suggestion
You can use memory limits parameters to control the total client memory usage,
```

File: docs/client-libraries-clients.md [hunk 2]
```diff
@@ -51,4 +51,36 @@ The following example creates a Python client using multiple advertised listener
...
+the producers and consumers under this client will compete for the memory assigned. See [PIP 74: Pulsar client memory limits](https://github.com/apache/pulsar/wiki/PIP-74%3A-Pulsar-client-memory-limits)
```
Comment (Anonymitaet) [inline docs/client-libraries-clients.md:59]:
```suggestion
the producers and consumers under this client will compete for the memory assigned. For implementation details, see [PIP 74: Pulsar client memory limits](https://github.com/apache/pulsar/wiki/PIP-74%3A-Pulsar-client-memory-limits).
```

Comment (Anonymitaet) [reply docs/client-libraries-clients.md:59]: Do we have any considerations/limitations that need to remind users?

**Review by Anonymitaet — COMMENTED** [review]
Can you attach the related code PR in the doc PR description (as commented in https://github.com/apache/pulsar-site/pull/565#issuecomment-1541064214)? So that reviewers will have more context, which can speed up the review process.

**Comment by Anonymitaet** [comment]
This PR adds doc for https://github.com/apache/pulsar-client-go/pull/991#issuecomment-1512391159

**Comment by shibd** [comment]
> Can you attach the related code PR in the doc PR description (as commented in https://github.com/apache/pulsar-site/pull/565#issuecomment-1541064214)? So that reviewers will have more context, which can speed up the review process.

oh, Sorry. Maybe we can change template?

### Preview



### Relation PR


### Documentation



- [ ] `doc` 
- [ ] `doc-required` 
- [ ] `doc-not-needed` 
- [ ] `doc-complete`

File: docs/client-libraries-clients.md (hunk 2 above)
Comment (shibd) [reply docs/client-libraries-clients.md:59]: I think nothing.

**Comment by Anonymitaet** [comment]
> > Can you attach the related code PR in the doc PR description (as commented in [#565 (comment)](https://github.com/apache/pulsar-site/pull/565#issuecomment-1541064214))? So that reviewers will have more context, which can speed up the review process.
> 
> oh, Sorry. Maybe we can change PR template?
> 
> ### Preview
> ### Relation PR
> ### Documentation
> * [ ]  `doc`
> * [ ]  `doc-required`
> * [ ]  `doc-not-needed`
> * [ ]  `doc-complete`

Sure, done in https://github.com/apache/pulsar-site/pull/572

**Review by Anonymitaet — APPROVED** [review]

**Pull request merged**